react/week2/todolist/src/components/Todos.js

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from "react";
+import { Timer } from './Timer';
+//import Done from "./Done";
+//import DeleteMe from "./DeleteMe";
+
+
+function Todos() {
+    const todos = [
+        {
+          id: 1,
+          description: "Get out of bed",
+        },
+        {
+          id: 2,
+          description: "Brush teeth",
+        },
+        {
+          id: 3,
+          description: "Eat breakfast",
+        },
+        {
+            id: 4,
+            description: "wash my car",
+          },
+      ];
+  
+  const [todoList, settodoList] = useState([]);
+  const [randomToDo, setRandomToDo] = useState(" ");
+  const [done, setDone] = useState(false);
+
+  function addTodoItem() {
+    let randomToDo = todos[Math.floor(Math.random() * todos.length)];
+    setRandomToDo(randomToDo);
+
+    settodoList(todoList.concat(randomToDo));
+  }
+
+   return (
+    <div className="App">      
+      <button onClick={addTodoItem}>Add to do</button>
+      <ul>
+        {todoList.map((todo) => {
+          return <div> 
+          <li key={todo.id}>{":"}{todo.description}</li> 
+          <button>done</button>     
+            <button>delete </button>     
```

Comment:
where is logic to delete todo list ?

Reply:
OKey,.thanks,.i will add it

Reply:
function deleteTodo(e) {
    const { id } = e.target.parentElement
    todoList.splice(id, 1)
    settodoList([...todoList]);
      }

Reply:
I have added it on the "reviews implented new PR",.Thanks